x-pack/distributions/internal/observabilitySRE/build-ext.gradle

```
@@ -1,17 +1,29 @@
 ext {
-    runTestsInFIPSMode = project.hasProperty('runTestsInFIPSMode') ? project.property('runTestsInFIPSMode').toBoolean() : false
+    fedrampHighMode = System.getenv('FEDRAMP_HIGH_MODE') == 'true' ||
+                      (project.hasProperty('fedrampHighMode') ? project.property('fedrampHighMode').toBoolean() : false)
```

Assuming that the `ORG_GRADLE_PROJECT_fedrampHighMode` works as documented, this can be made a little less redundant with:

```suggestion
    fedrampHighMode = Objects.requireNonNullElse(project.findProperty('fedrampHighMode'), false).toBoolean()
```